alacritty/src/input.rs

```diff
@@ -341,17 +341,19 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
             self.update_selection_scrolling(y);
         }
 
+        let display_offset = self.ctx.terminal().grid().display_offset();
+        let old_point = self.ctx.mouse().point(&size_info, display_offset);
```

Comment:
It's kinda unfortunate that we have to calculate the old point here just to detect if the cursor's cell has changed.